blog/templates/blog/article_list.html

```diff
@@ -17,7 +18,9 @@ <h1>Your Articles</h1>
         <h1>All Articles</h1>
         <ul>
             {% for article in blogs %}
```

Comment:
dapat by category to, refer to the figma and the specs so this should be 
`{% for category in categories %}`
then may sub loop na `{% for blog in category.blogs %}`